### DISCUSSION TITLE
Grayscale Output on Mac SE/30

### DISCUSSION THREAD

**Discussion by hemulen242** [akuker/RASCSI#1168] [Ideas]
Reading a thread on the 68kmla forums on making a grayscale card for SE/30, would it be possible to send the HDMI video produced by the virtual SCSI video adapter, via a newly designed neck board on the CRT?

**Comment by rdmark** [comment]
This should be possible if you add circuitry to convert the digital video signal to RGB.

However I wouldn't recommend leveraging SCSI video for this purpose, since the latency of the SCSI bus + GPIO is severe. At low resolutions and color depths it might be acceptable as a secondary display, but using it for a primary display isn't advisable.

Just my two cents!
